apps/site/assets/js/test/trip-plan_test.js

```diff
@@ -95,4 +96,117 @@ describe("trip-plan", () => {
       assert.equal($from_lng.val(), 4);
     });
   });
+
+  describe("TripPlannerLocControls", () => {
```

Comment:
Would it make sense for this to live in a `trip-planner-location-controls_test.js` in order to match the structure of the implementation file?

Reply:
It would! But `TripPlannerTimeControls` is also tested in this file instead of a standalone one, so, chose to follow that.